src/features/playground/shareLinks/ShareLinkState.tsx

```diff
@@ -0,0 +1,13 @@
+type ShareLinkState = {
+  isFolder: string;
+  tabs: string;
+  tabIdx: string;
+  chap: string;
+  variant: string;
+  ext: string;
+  exec: string;
+  files: string;
+  prgrm: string;
+};
```

Comment:
Nit: instead of using `Partial<ShareLinkState>`, `ShareLinkState` itself should be the partial of the object.

Though it's the same in terms of the actual type, semantically, the there is no partial share link state. The share link state itself is the partial type, if you get what I mean.

Reply:
changed to `type ShareLinkState = {...};`